js/components/ChooseAvatar/Avatar.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import {
+  Text,
+  View,
+  Modal,
+  TouchableOpacity,
+  StyleSheet,
+  Image
+} from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import globalStyles from "../../config/styles";
+import styles from "./styles";
+
+export default class Avatar extends Component {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
remove constructor. You are not creating any state.

Reply:
done

Reply:
It's not done.

Reply:
sorry I didnt push before you checked again. they are there